tests/system/providers/google/marketing_platform/example_display_video.py

```diff
@@ -0,0 +1,140 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG that shows how to use DisplayVideo.
+"""
+import os
+from datetime import datetime
+from typing import Dict
+
+from airflow import models
+from airflow.providers.google.marketing_platform.hooks.display_video import GoogleDisplayVideo360Hook
+from airflow.providers.google.marketing_platform.operators.display_video import (
+    GoogleDisplayVideo360CreateReportOperator,
+    GoogleDisplayVideo360DeleteReportOperator,
+    GoogleDisplayVideo360DownloadReportOperator,
+    GoogleDisplayVideo360RunReportOperator,
+)
+from airflow.providers.google.marketing_platform.sensors.display_video import (
+    GoogleDisplayVideo360ReportSensor,
+)
+from airflow.utils.trigger_rule import TriggerRule
+
+# [START howto_display_video_env_variables]
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+DAG_ID = "example_display_video"
+BUCKET = os.environ.get("GMP_DISPLAY_VIDEO_BUCKET", "gs://INVALID BUCKET NAME")
+ADVERTISER_ID = os.environ.get("GMP_ADVERTISER_ID", 1234567)
+OBJECT_NAME = os.environ.get("GMP_OBJECT_NAME", "files/report.csv")
+PATH_TO_UPLOAD_FILE = os.environ.get("GCP_GCS_PATH_TO_UPLOAD_FILE", "test-gcs-example.txt")
+PATH_TO_SAVED_FILE = os.environ.get("GCP_GCS_PATH_TO_SAVED_FILE", "test-gcs-example-download.txt")
+BUCKET_FILE_LOCATION = PATH_TO_UPLOAD_FILE.rpartition("/")[-1]
+SDF_VERSION = os.environ.get("GMP_SDF_VERSION", "SDF_VERSION_5_1")
+BQ_DATA_SET = os.environ.get("GMP_BQ_DATA_SET", "airflow_test")
+GMP_PARTNER_ID = os.environ.get("GMP_PARTNER_ID", 123)
+ENTITY_TYPE = os.environ.get("GMP_ENTITY_TYPE", "LineItem")
+ERF_SOURCE_OBJECT = GoogleDisplayVideo360Hook.erf_uri(GMP_PARTNER_ID, ENTITY_TYPE)
```

Comment:
The amount of the env variables is one of the problems with the old design. When migrating to the new design it would be best to reduce required env variables rewriting them to constants or fstring reusing ENV_ID and DAG_ID whenever needed. The example of removing the need of environment variable can be seen in this PR: https://github.com/apache/airflow/pull/23282/files#diff-8d21adbd43cba4bdb6175bd7fcc99cc7352a7f2788c9b3486a86148985950a6aL55